src/pudl/metadata/fields.py

```diff
@@ -263,6 +263,10 @@
         "description": "Average monthly coincident peak (CP) demand (for requirements purchases, and any transactions involving demand charges). Monthly CP demand is the metered demand during the hour (60-minute integration) in which the supplier's system reaches its monthly peak. In megawatts.",
         "unit": "MW",
     },
+    "combined_heat_power": {
```

Comment:
Is this really a different field than `associated_combined_heat_power` or do they represent the same characteristic applied to generators vs. boilers?